app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class SessionsController < ApplicationController
+  def new
+
+    if current_user.role == 1
+      flash[:notice] = "You're already logged in."
+      redirect_to '/profile'
+    elsif current_user.role == 2
+      flash[:notice] = "You're already logged in."
+      redirect_to '/merchant/dashboard'
+    elsif current_user.role == 3
+      flash[:notice] = "You're already logged in."
+      redirect_to '/admin/dashboard'
+    end
```

Comment:
If we have time to refactor do you think we can make a helper method based on this and the if block in SessionsController create